jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml

```diff
@@ -21,7 +21,7 @@
         - gcs-uploader
     provider-env: |
         export KUBERNETES_PROVIDER="aws"
-        export E2E_MIN_STARTUP_PODS="1"
+        export E2E_MIN_STARTUP_PODS="8"
```

Comment:
it might be worth replicating the comment describing where this number came from.

long-term I wonder if we should have a smarter way to list the pods we expect so we don't have to try to keep a context-free number up-to-date.